build/charts/antrea-windows/templates/daemonset.yaml

```diff
@@ -0,0 +1,143 @@
+        app: antrea
+        component: antrea-agent
+    spec:
+      containers:
+      - args:
+        - -file
+        - $env:CONTAINER_SANDBOX_MOUNT_POINT/var/lib/antrea-windows/Run-AntreaAgent.ps1
+        command:
+        - powershell
+        env:
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+        imagePullPolicy: IfNotPresent
+        {{- if .Values.includeOVS }}
+        lifecycle:
+          postStart:
+            exec:
+              command:
+              - powershell
+              - -file
+              - $env:CONTAINER_SANDBOX_MOUNT_POINT/var/lib/antrea-windows/VMSwitchExtension-AntreaAgent.ps1
+              - -VMSwitchExtension
+              - enable
+          preStop:
+            exec:
+              command:
+              - powershell
+              - -file
+              - $env:CONTAINER_SANDBOX_MOUNT_POINT/var/lib/antrea-windows/VMSwitchExtension-AntreaAgent.ps1
+              - -VMSwitchExtension
+              - disable
+        {{- end}}
+        name: antrea-agent
+        volumeMounts:
+        - mountPath: /etc/antrea
+          name: antrea-windows-config
+        - mountPath: /var/lib/antrea-windows
+          name: antrea-agent-windows
+        - mountPath: /var/log/antrea/
+          name: var-log-antrea
+      {{- if .Values.includeOVS }}
+      - args:
+        - -file
+        - $env:CONTAINER_SANDBOX_MOUNT_POINT/var/lib/antrea-windows/Run-AntreaOVS.ps1
+        command:
+        - powershell
+        image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+        imagePullPolicy: IfNotPresent
+        name: antrea-ovs
+        volumeMounts:
+        - mountPath: /var/lib/antrea-windows
+          name: antrea-agent-windows
+        - mountPath: /var/log/openvswitch
+          name: var-log-antrea
+          subPath: openvswitch
+      {{- end}}
+      hostNetwork: true
+      initContainers:
+      - args:
+        - -file
+        - $env:CONTAINER_SANDBOX_MOUNT_POINT/var/lib/antrea-windows/Install-WindowsCNI.ps1
+        command:
+        - powershell
+        image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+        imagePullPolicy: IfNotPresent
+        name: install-cni
+        volumeMounts:
+        - mountPath: /etc/antrea
+          name: antrea-windows-config
+          readOnly: true
+        - mountPath: /var/lib/antrea-windows
+          name: antrea-agent-windows
+      {{- if .Values.includeOVS }}
+      - args:
+        - -file
+        - $env:CONTAINER_SANDBOX_MOUNT_POINT/var/lib/antrea-windows/Install-OVSDriver.ps1
+        command:
+        - powershell
+        image: antrea/antrea-windows:latest
```

Comment:
Could you update this image value to align with the others? This bug was introduced by a previous PR, but it would be better if we could fix it in this PR. The GitHub CI didn't catch this bug because when running generate-manifest-windows.sh, it works well in the dev mode but only fails in the release mode. cc @antoninbas

Reply:
Thanks for catching this @XinShuYang 
@shikharish please replace with `{{ .Values.image.repository }}:{{ .Values.image.tag }}`